models/Asset.php

```diff
@@ -415,6 +415,7 @@ public static function getList($config = [])
 
         $listClass = Listing::class;
 
+        /** @var Listing $list */
```

Comment:
this has nothing to do with this PR

Reply:
Without this PhpStan fails because `self::getModelFactory()->build($listClass)` returns `AbstractModel`

Reply:
I see